app/components/DataTable/components/KbMatchesActionCellRenderer/index.tsx

```diff
@@ -32,9 +34,18 @@ const KbMatchesActionCellRenderer = (props: ActionCellRendererProps) => {
   const [therapeuticTargetType, setTherapeuticTargetType] = useState<TherapeuticTargetType>(null);
   const [isLoading, setIsLoading] = useState(false);
   const [iprEvidenceLevels, setIprEvidenceLevels] = useState(null);
+  const [isClinicalTrial, setIsClinicalTrial] = useState(false);
 
   const isMult = Array.isArray(kbStatementId);
 
+  // Filtering whether a statement is clinical trial using context and recruitment status
+  useEffect(() => {
```

Comment:
alternative is to use `useMemo` instead of `useState` with `useEffect`

Reply:
will it make it more efficient? if so I will switch over to useMemo!

Reply:
useMemo is supposed to be used for expensive calculations, in which case the `.includes` might fit the requirement ... though actual numbers you'd have to run to see what's up...

but the more practical reason is to have less lines, at the cost of negligible performance hits, if any